js/express/customization/customization.utils.js

```diff
@@ -40,6 +40,17 @@ const addBlankRowIfRequired = (viewDiv, addButton) => {
   }
 };
 
+const mappingAlreadyExists = (mappingData, mapping, view) => mappingData.some((md) => {
+  if (md.mapping === view && md.id !== mapping.id) {
+    let duplicate = true;
+    customizedFields[view].forEach((attr) => {
```

Comment:
Would `every` or `some` work here instead of `forEach` since those functions are meant to return a boolean based off of some condition.